client/src/Controller/Ndr/NdrController.php

```diff
@@ -105,11 +110,14 @@ public function indexAction(Redirector $redirector)
 
         $clients = $user->getClients();
         $client = !empty($clients) ? $clients[0] : null;
-        $coDeputies = !empty($client) ? $this->getCoDeputiesForClient($user) : [];
+
+        $clientWithCoDeputies = $this->restClient->get('client/' . $client->getId(), 'Client', ['client', 'client-users', 'user']);
```

Comment:
Could we replace this with:

```suggestion
        $clientWithCoDeputies = $this->clientApi->getWithUsersV2($client->getId()));
```

Trying to move restclient calls into the appropriate API classes when we come across them.